packages/salesforcedx-vscode-lwc/src/messages/i18n.ts

```diff
@@ -21,6 +21,8 @@ export const messages = {
   force_lightning_lwc_start_text: 'SFDX: Start Local Development Server',
   force_lightning_lwc_start_not_found:
     'To run this command, first install the @salesforce/lwc-dev-server plugin. For more info, see [https://developer.salesforce.com/docs/component-library/documentation/lwc/lwc.get_started_local_dev](https://developer.salesforce.com/docs/component-library/documentation/lwc/lwc.get_started_local_dev).',
+  force_lightning_lwc_start_addr_in_use:
```

Comment:
@hbc-bindu Please let me know if you'd like to update this text. I was debating about adding more information on how to kill the process, but not sure if we should actually include that piece. It's also a long message already.

Reply:
@sfsholden, very minor changes:
The local development server couldn’t start as the address is already in use. To resolve, do one of these: \n1) Stop the local dev server running on any another instance.\n 2) Change the default port [Configuration for Projects (Optional)](https://developer.salesforce.com/docs/component-library/documentation/en/lwc/lwc.get_started_local_dev_setup).\n 3) Kill the process running on the specified port.